examples/dynamic_execution_example.py

```diff
@@ -1,7 +1,9 @@
-def dynamic_execution(name):
-    """Append in front of string 'name'"""
-    eval('hello, my name is ' + name)
+def get_power_function(power):
+    """Return a function which takes its argument to the <power>th power."""
+    func = eval("lambda x: x ** {}".format(power))
+    return func
 
-if __name__ == '__main__':
-    dynamic_execution('hayley')
 
+if __name__ == '__main__':
```

Comment:
The changed example is good. But in this case, I don't need to see the main block at all - the error should be detected just in the function definition.